server/api.go

```diff
@@ -145,6 +153,28 @@ func (a *API) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	a.router.ServeHTTP(w, r)
 }
 
+func (a *API) ServeHTTPWithMetrics(w http.ResponseWriter, r *http.Request) {
```

Comment:
I'm curious why this isn't still a middleware?

Reply:
Done